docs/_layouts/docs.html

```diff
@@ -29,7 +29,7 @@ <h1 class="bd-title" id="content">{{ page.title }}</h1>
           {{ content }}
```

Comment:
wondering if this may not be the more appropriate location for `<main>` (depends if we count the side navigation as content or not). if so, i'd suggest keeping the `<div class="col-12...">` and wrapping its contents in a `<main role="main">`, rather than replacing the `<div>` altogether